### PR TITLE
chore: tweak output & verbose messages

### PR DESCRIPTION
**Pull request by zroubalik**
Signed-off-by: Zbynek Roubalik <zroubali@redhat.com>

Outputs of commands were quite incositent, trying to fix that.

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -414,11 +420,17 @@ func (c *Client) Deploy(path string) (err error) {
 	}
 
 	// Push the image for the named service to the configured registry
+	if c.verbose {
+		fmt.Println("\nPushing Function image to the registry:")
```
Comment (lance) [inline client.go:424]: Why the ":" at the end of this message (as well as below on 432)?

File: cmd/describe.go [hunk 2]
```diff
@@ -109,25 +110,43 @@ func newDescribeConfig(args []string) describeConfig {
...
 func (d description) Human(w io.Writer) error {
-	fmt.Fprintln(w, d.Name)
+	fmt.Fprintln(w, "Function name:")
+	fmt.Fprintf(w, "  %v\n", d.Name)
+	fmt.Fprintln(w, "Function is built in image:")
+	fmt.Fprintf(w, "  %v\n", d.Image)
+	fmt.Fprintln(w, "Function is deployed as Knative Service:")
+	fmt.Fprintf(w, "  %v\n", d.KService)
+	fmt.Fprintln(w, "Function is deployed in namespace:")
+	fmt.Fprintf(w, "  %v\n", d.Namespace)
```
Comment (lance) [inline cmd/describe.go:120]: Much nicer!

File: cmd/describe.go [hunk 3]
```diff
@@ -109,25 +110,43 @@ func newDescribeConfig(args []string) describeConfig {
...
-	return d.Plain(w)
+	return nil
 }
 
 func (d description) Plain(w io.Writer) error {
 	fmt.Fprintf(w, "NAME %v\n", d.Name)
+	fmt.Fprintf(w, "IMAGE %v\n", d.Image)
+	fmt.Fprintf(w, "KNATIVE SERVICE %v\n", d.KService)
+	fmt.Fprintf(w, "NAMESPACE %v\n", d.Namespace)
+
```
Comment (lance) [inline cmd/describe.go:141]: Why all caps here (and below)? I know it's what was there before, but why?

**Review by lance — COMMENTED** [review]
A couple of minor comments/questions. Otherwise LGTM.

File: client.go (hunk 1 above)
Comment (zroubalik) [reply client.go:424]: I can remove that, but the output of `docker push/build` follows these lines

Comment (lance) [reply client.go:424]: Oh that makes sense then

File: cmd/describe.go (hunk 3 above)
Comment (lance) [reply cmd/describe.go:141]: I think let's be consistent. Just a suggestion.

Comment (lance) [reply cmd/describe.go:141]: Oh sorry that you had to wait 2 hours for a simple answer... Yes I meant remove all caps to be consistent with other output strings.

**Review by lance — APPROVED** [review]

**Pull request merged**